google/auth/transport/requests.py

```diff
@@ -36,10 +36,10 @@
     )
 import requests.adapters  # pylint: disable=ungrouped-imports
 import requests.exceptions  # pylint: disable=ungrouped-imports
-from requests.packages.urllib3.util.ssl_ import (
+import six  # pylint: disable=ungrouped-imports
+from urllib3.util.ssl_ import (
```

Comment:
This change to move from `requests.packages.urllib3` to `urllib3` is required to resolve a mypy failure which started in `types-requests==2.27.5`. See
https://github.com/python/typeshed/issues/6893

In `requests` version 2.16.0 and onwards, `requests.packages.urllib3` is the same as `urllib3`
https://github.com/psf/requests/pull/4067